src/Entity/Tricks.php

```diff
@@ -16,6 +17,35 @@
  */
 class Tricks
 {
+    /**
+     * @ORM\OneToMany(targetEntity="App\Entity\Comment", mappedBy="tricks", cascade={"persist", "remove"})
+     * @Assert\Count(
+     *      min = 1,
+     *      max = 5,
+     *      minMessage = "tricks.images.min",
+     *      maxMessage = "tricks.images.max"
+     * )
+     * @ORM\JoinColumn(nullable=true)
+     * @Assert\Valid()
+     */
+    private $images;
+
+    /**
+     * @return mixed
+     */
+    public function getImages()
+    {
+        return $this->images;
+    }
+
+    /**
+     * @param mixed $images
+     */
+    public function setImages($images): void
+    {
+        $this->images = $images;
```

Comment:
Essais de prendre l'habitude de retourner $this dans tes setter.
Cela permet d'enchainer les setter
``` $this->setName('toto')->setImages(...)....